projects/libcoap/pdu_parse_target.c

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
best to move this in your github repo. See https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md. fine as a first step here. upto you, you can move now or later.